lib/docker.ts

```diff
@@ -137,12 +137,24 @@ class Docker {
     return await dockerPull.pull(registry, repo, tag, opt);
   }
 
+  public async isDockerExperimentalEnabled(): Promise<boolean> {
+    const output = await subProcess.execute("docker system info");
```

Comment:
Interesting! Isn't every argument its own element of a list, for example
```typescript
await subProcess.execute("docker", ["system", "info"]);
```

Reply:
Ah you're right, I think that is the way the arguments should be passed!